packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -136,6 +180,62 @@ contract DeployConfig is Script {
         preimageOracleMinProposalSize = stdJson.readUint(_json, "$.preimageOracleMinProposalSize");
         preimageOracleChallengePeriod = stdJson.readUint(_json, "$.preimageOracleChallengePeriod");
         preimageOracleCancunActivationTimestamp = stdJson.readUint(_json, "$.preimageOracleCancunActivationTimestamp");
+
+        //////////////////////////////////////////////////////
+        /// Genesis Config Properties
+        //////////////////////////////////////////////////////
+        l2GenesisBlockNumber = parseJsonUintWithDefault("$.l2GenesisBlockNumber", uint256(0));
+        l2GenesisCanyonTimeOffset = parseJsonUintWithDefault("$.l2GenesisCanyonTimeOffset", uint256(0));
+
+        homesteadBlock = parseJsonUintWithDefault("$.homesteadBlock", uint256(0));
+        eip150Block = parseJsonUintWithDefault("$.eip150Block", uint256(0));
+        eip155Block = parseJsonUintWithDefault("$.eip155Block", uint256(0));
+        eip158Block = parseJsonUintWithDefault("$.eip158Block", uint256(0));
+        byzantiumBlock = parseJsonUintWithDefault("$.byzantiumBlock", uint256(0));
+        constantinopleBlock = parseJsonUintWithDefault("$.constantinopleBlock", uint256(0));
+        petersburgBlock = parseJsonUintWithDefault("$.petersburgBlock", uint256(0));
+        istanbulBlock = parseJsonUintWithDefault("$.istanbulBlock", uint256(0));
+        muirGlacierBlock = parseJsonUintWithDefault("$.muirGlacierBlock", uint256(0));
+        berlinBlock = parseJsonUintWithDefault("$.berlinBlock", uint256(0));
+        londonBlock = parseJsonUintWithDefault("$.londonBlock", uint256(0));
+        arrowGlacierBlock = parseJsonUintWithDefault("$.arrowGlacierBlock", uint256(0));
+        grayGlacierBlock = parseJsonUintWithDefault("$.grayGlacierBlock", uint256(0));
+        mergeNetsplitBlock = parseJsonUintWithDefault("$.mergeNetsplitBlock", uint256(0));
+
+        shanghaiTime = parseJsonUintWithDefault("$.shanghaiTime", uint256(0));
+        cancunTime = parseJsonUintWithDefault("$.cancunTime", uint256(0));
+        bedrockBlock = parseJsonUintWithDefault("$.bedrockBlock", uint256(0));
+        regolithTime = parseJsonUintWithDefault("$.regolithTime", uint256(0));
+        canyonTime = parseJsonUintWithDefault("$.canyonTime", uint256(0));
+        ecotoneTime = parseJsonUintWithDefault("$.ecotoneTime", uint256(0));
+        interopTime = parseJsonUintWithDefault("$.interopTime", uint256(0));
+
+        terminalTotalDifficulty = parseJsonUintWithDefault("$.terminalTotalDifficulty", uint256(0));
+        try vm.parseJsonBool(_json, "$.terminalTotalDifficultyPassed") returns (bool data) {
+            terminalTotalDifficultyPassed = data;
+        } catch {
+            terminalTotalDifficultyPassed = true;
+        }
+
+        eip1559Elasticity = parseJsonUintWithDefault("$.eip1559Elasticity", uint256(0));
+        eip1559Denominator = parseJsonUintWithDefault("$.eip1559Denominator", uint256(0));
```

Comment:
These should not default to 0, there are default values for them per the protocol, perhaps in op-geth they are defined

Reply:
Updated to as [per this](https://github.com/ethereum-optimism/optimism/blob/9cca805d5e46746c60070d12bec94e1d93c4dcf3/op-chain-ops/genesis/genesis.go#L30-L41)

```
eip1559Elasticity = parseJsonUintWithDefault("$.eip1559Elasticity", uint256(10));
eip1559Denominator = parseJsonUintWithDefault("$.eip1559Denominator", uint256(50));
eip1559DenominatorCanyon = parseJsonUintWithDefault("$.eip1559DenominatorCanyon", uint256(250));
```